src/shared/domain/definition/jurisdiction-config.model.ts

```diff
@@ -0,0 +1,4 @@
+export class JurisdictionConfig {
```

Comment:
should we have named this as JurisdictionUIConfig, as we named it so elsewhere? -- to avoid confusion with a more generic sense of jurisdiction configuration by definition files.

Reply:
Addressed 👍